src/main/java/de/zalando/aruha/nakadi/repository/kafka/KafkaTopicRepository.java

```diff
@@ -49,7 +49,7 @@ public KafkaRepository(final ZooKeeperHolder zkFactory, final KafkaFactory kafka
     }
 
     @Override
-    public List<Topic> listTopics() throws NakadiException {
+    public List<Topic> listTopics() throws ServiceUnavailableException {
```

Comment:
We had an offline discussion about this and came to the conclusion that we want to use a 503 if it doesn't work because an underlying resource is not available. It emphasises that it's probably a temporary problem and it might be worth to retry after a certain time frame.